BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift

```diff
@@ -77,6 +77,10 @@ struct AddEditSendItemView: View { // swiftlint:disable:this type_body_length
                         )
                     )
                 }
+
+                cancelToolbarButton {
+                    store.send(.dismissPressed)
+                }
```

Comment:
🤔 Is it ok for the cancel button to be to the right of the avatar in the share extension? I feel like the cancel button should be the first thing on the toolbar to the left; like in `VaultAutofillListView`.
| (Cancel) (Avatar)     Title     (Other actions) |

Reply:
Good catch! I thought I found all of those, but I guess not!

I guess we don't have any snapshot tests of the share extension, which is how I missed this, I think. I'll swing back around and maybe try to get some more snapshots in.

Reply:
Awesome thanks!